<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="totalPressLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Presses: </string>
    </property>
   </widget>
   <widget class="QLabel" name="rateLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Presses Per Second: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Elapsed: </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
